package Protocol;
message MessageBase
{
    required int32 opcode = 1;
}